src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Runtime/CSharpAvoidZeroLengthArrayAllocationsFixer.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.NetCore.Analyzers.Runtime;
+
+namespace Microsoft.NetCore.CSharp.Analyzers.Runtime
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, LanguageNames.VisualBasic), Shared]
+    public sealed class CSharpAvoidZeroLengthArrayAllocationsFixer : AvoidZeroLengthArrayAllocationsFixer
+    {
+        protected override T AddElasticMarker<T>(T syntaxNode)
+            => syntaxNode.WithTrailingTrivia(syntaxNode.GetTrailingTrivia().Add(SyntaxFactory.ElasticMarker));
```

Comment:
This is only needed in VB, but I'm calling it for both languages for simplicity.

Without this, `End Sub` isn't placed on its own line. I think this might be a SyntaxGenerator bug on roslyn side? cc @CyrusNajmabadi